test/fixtures/roles.sql

```diff
@@ -17,7 +17,7 @@ ALTER ROLE postgrest_test_authenticator SET pgrst."db-tx-end" = 'commit-allow-ov
 ALTER ROLE postgrest_test_authenticator SET pgrst."db-schemas" = 'test, tenant1, tenant2';
 ALTER ROLE postgrest_test_authenticator SET pgrst."db-root-spec" = 'root';
 ALTER ROLE postgrest_test_authenticator SET pgrst."db-prepared-statements" = 'false';
-ALTER ROLE postgrest_test_authenticator SET pgrst."db-pre-request" = 'custom_headers';
+ALTER ROLE postgrest_test_authenticator SET pgrst."db-pre-request" = 'test.custom_headers';
```

Comment:
Had to add the schema qualifier because when switching to the v1 schema there's no `custom_headers` func, which have errors.

Btw, thanks to this I noted here that the `root-spec` doesn't support a schema qualifier. Which looks necessary when having multiple schemas.

Reply:
Are you tracking the missing schema qualifier for root-spec somewhere?

Reply:
Mentioned the issue on https://github.com/PostgREST/postgrest/issues/1731. Didn't want to open a full issue as bug because as it is, `root-spec` is incomplete.